src/utils/formatedDate.js

```diff
@@ -0,0 +1,10 @@
+export const formatedDate = (date) =>
+  date
+    .toLocaleString("en", {
+      day: "numeric",
+      month: "numeric",
+      year: "2-digit",
+    })
+    .split(",")[0]
+    .split("/")
+    .join(".")
```

Comment:
we should utilise the built-in date formatter in this case.
Check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) out